backend/EasySpeak.Core/EasySpeak.Core.DAL/Migrations/20230409121118_ChangeAllDefaultValuesByExisting.cs

```diff
@@ -10,11 +10,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
         {
             migrationBuilder.Sql(@"UPDATE Users
                 SET Country = 1
-                WHERE Country = 0;
-
-                UPDATE Lessons
-                SET CreatedBy = (SELECT MIN(Id) FROM Users)
-                WHERE CreatedBy IS NULL;");
```

Comment:
It won't work if you decided to change migration, it will stay the same on current db version

Reply:
And can you check that the next migration is empty?

Reply:
I mean, if you generate one more